Cannot compile in macos, projectPath missing for rooibos when making

Trying to compile and deploy on macos, but, it is not getting past make test, getting the error:

```
make test
*** Preparing Staging Area ***
  >> removing old application zip ./out/apps/Jellyfin_Roku.zip
  >> creating destination directory ./out/apps
  >> setting directory permissions for ./out/apps
  >> creating destination directory ./out/staging
  >> moving application to ./out/staging
Processing....
parsing config { testPath: 'out/staging/source/tests/specs',
  rootPath: '',
  outputPath: 'out/staging/source/tests/rooibos' }
/usr/local/lib/node_modules/rooibos-preprocessor/dist/lib/ProcessorConfig.js:21
        throw new Error('Config does not contain projectPath property' + docsLink);
        ^

Error: Config does not contain projectPath property
Please read the docs for usage details https://github.com/georgejecook/rooibos/blob/master/docs/index.md#rooibosc
    at Object.createProcessorConfig (/usr/local/lib/node_modules/rooibos-preprocessor/dist/lib/ProcessorConfig.js:21:15)
    at Command.<anonymous> (/usr/local/lib/node_modules/rooibos-preprocessor/dist/cli.js:46:45)
    at Command.listener (/usr/local/lib/node_modules/rooibos-preprocessor/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:182:13)
    at Command.parseArgs (/usr/local/lib/node_modules/rooibos-preprocessor/node_modules/commander/index.js:665:12)
    at Command.parse (/usr/local/lib/node_modules/rooibos-preprocessor/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/rooibos-preprocessor/dist/cli.js:51:9)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
make: *** [prep_tests] Error 1
```

--- a/tests/.rooibosrc.json
+++ b/tests/.rooibosrc.json
@@ -1,5 +1,6 @@
 {
     "testPath": "out/staging/source/tests/specs",
     "rootPath": "",
-    "outputPath": "out/staging/source/tests/rooibos"
+    "outputPath": "out/staging/source/tests/rooibos",
+    "projectPath": "."
   }

